signrelease.sh

```diff
@@ -82,7 +85,7 @@ for ext in $ARCHIVE_TYPES; do
     fi
 
     # sign archive
-    gpg --armor --detach-sign "$TMP_DIR/${project}-${tag}.${ext}"
+    gpg -u "${originGitHub##*/} release" --armor --detach-sign "$TMP_DIR/${project}-${tag}.${ext}"
```

Comment:
So this is the naming part?

Well… that is not much generic. It's just a coincidence that we chose this name for PrivateBin? 

Maybe better use that part up in the changable constants/variables at the top of the script? (If you want using a placeholder that gets replaced later like `GITHUB_REPO_NAME` or so?

Reply:
It's not a coincidence: I specifically labelled my key "PrivateBin release" and the above change codifies this. It is generic in that you can have a key per repo and it must be named "<Repo name> release". You can rename your local keys in the key ring to ensure none of your other keys get picked. I'm not aware of an easy way to have multiple key rings per user and tell gpg which ring to use...